resources/views/inc/menu_user_dropdown.blade.php

```diff
@@ -1,9 +1,9 @@
 <div class="nav-item dropdown">
     <a href="#" class="nav-link d-flex lh-1 text-reset p-0" data-bs-toggle="dropdown" aria-label="Open user menu">
-        <span class="avatar avatar-sm" style="background-image: url({{ app(Creativeorange\Gravatar\Gravatar::class)->get(backpack_user()->email) }})"></span>
+        <span class="avatar avatar-sm" style="background-image: url({{ app(Creativeorange\Gravatar\Gravatar::class)->get(backpack_user()->email, ['size' => 80]) }})"></span>
```

Comment:
This makes the avatar look crispy on devices with high DPI.
Basically, the image size is `32px`, I opted for `80px` since it is 2.5 times the actual size. So this will support devices with a 1:2.5 pixel aspect ration.

---

**The reason this is here and not on config.** 
The size of the gravatar image should be related with the place where it's being used. Other places may need it bigger, other may need it smaller. IMO placing it here is the safer way, anyone customizing the theme may change the value specifically here.